### ISSUE TITLE
not working on wsl2

### ISSUE THREAD

**Issue by prettydeep** [dfaker/stable-diffusion-webui-cv2-external-masking-script#4]
opencv-python is installed. the webui runs fine and I can select the script. But when I click the generate button the masking window does not appear and no image iterations take place. i've tried defining the $DISPLAY variable with every possible address, but still nothing works.

**Comment by dfaker** [comment]
Slipped me by that it has a GUI capability now, I take it we have nothing coming up in the terminal as an error?

What does something super minimal like:
```python
import cv2
import numpy as np
img = np.random.random((500,500,3))
cv2.imshow('rand',img)
cv2.waitKey(-1)
```
do, nothing at all?

**Comment by prettydeep** [comment]
Correct, the terminal shows no errors.

I tried the code you posted above and got the same result, the terminal shows nothing. However, I did some digging and found [this](https://github.com/microsoft/WSL/issues/6430#issuecomment-881766991). When I set `export DISPLAY=:0`, it worked....

<img width="288" alt="test" src="https://user-images.githubusercontent.com/77951303/193824833-22629716-0992-4f5f-b405-07579d78f10b.png">

I tried the masking script again within the stable diffusion webui, after setting `export DISPLAY=:0`, and it worked. 

Sorry, guess this was a problem on my end. Thanks for the help.
